docs/Cloud_integrations/Device_Integration_Center/webhooks.md

```diff
@@ -20,19 +20,15 @@ The **webhook event handler** delivers HTTP(S) requests to public endpoints. Web
 
 To create a new webhook event handler, select: **Integrations** > **Data Integration Center**.
 
-![Data integration center](images/data-integration-center.png "Data Integration Center")
+![Data integration center](images/data-integration-center.webp "Data Integration Center")
 
-Select **Add event handler**.
+Click **Add event handler** and select **Webhook**.
 
-![Add event handler](images/add-event-handler.png "Add event handler")
-
-Select **Webhook**.
-
-![Add webhook](images/webhook.png "Add webhook")
+![Add event handler](images/add-webhook-handler.webp "Add event handler"){ width=25% }
 
 Provide **basic information** such as the name and description, optionally enabling the option to activate the event handler after creation.  
```

Comment:
Provide **basic information** such as the name and description
!!! info
You can activate the event handler after creation.